randovania/games/common/prime_family/gui/export_validator.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from pathlib import Path
+
+import nod
+
+from randovania.gui.dialog.game_export_dialog import is_file_validator
+
+
+def discover_game(iso_path: Path) -> tuple[str, str] | None:
+    if not iso_path.is_file():
+        return None
+
+    try:
+        disc, is_wii = nod.open_disc_from_image(iso_path)
+        data_partition: nod.Partition = disc.get_data_partition()
+        header: nod.DolHeader = data_partition.get_header()
+    except RuntimeError as e:
+        logging.warning("Unable to parse %s: %s", str(iso_path), str(e))
+        return None
+
+    return header.game_id.decode("UTF-8"), header.game_title.split(b"\x00")[0].decode("UTF-8")
+
+
+def is_prime1_iso_validator(file: Path | None) -> bool:
+    """Returns False when the given path exists and is either not an ISO or a Prime 1 ISO, True otherwise"""
+    if is_file_validator(file):
+        return True
+
+    if file.suffix.lower() == ".iso":
```

Comment:
is there a reason this line isn't in the echoes verifier?

Reply:
Echoes only supports iso, but randomprime supports a few extra formats that aren't supported by nod.

Reply:
might wanna add a comment mentioning that limitation then